Sources/Broadcast/BroadcastEdge.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+
+extension Warp {
+    /**
+        Enum describing  which edge the Broadcast view will appear from/disappear to
+     */
+    public enum BroadcastEdge: CaseIterable {
+        case top
+        case bottom
+
+        public var asAlignment: Alignment {
```

Comment:
Should we have this property as public?
Also when we have it inside the domain itself, we are glueing ui and logic together,
Same as next computed property `asEdge`.

Reply:
Since I copy and pasted this from Toast which @pbodsk wrote I am not gonna play with it now. We can discuss this later and see what's best.